src/main/java/co/novu/common/base/Novu.java

```diff
@@ -878,12 +878,12 @@ public TenantResponse updateTenant(TenantRequest request, String identifier) {
         }
     }
 
-    public DeleteTenantResponse deleteTenant(String identifier) {
+    public void deleteTenant(String identifier) throws IOException, NovuNetworkException {
```

Comment:
This cannot return void. It has to return DeleteTenantResponse

Reply:
@mayorJAY updated